jason/demo/chart/medea-demo/values.yaml

```diff
@@ -29,6 +29,8 @@ server:
       db:
         redis:
           db_number: 0
+      cli:
+        port: 5766
```

Comment:
You should bump up Chart versionaccroding to semver, whenever you change its sources.